Makefile

```diff
@@ -88,7 +88,7 @@ kind-load: ## Load-image loads the currently constructed image onto the cluster
 	${KIND} load docker-image $(IMAGE) --name $(KIND_CLUSTER_NAME)
 
 kind-cluster: ## Standup a kind cluster for e2e testing usage
-ifeq (1, $(shell kind get clusters | grep ${KIND_CLUSTER_NAME} | wc -l))
+ifeq (1, $(shell kind get clusters | grep ${KIND_CLUSTER_NAME} | wc -l | xargs))
```

Comment:
What's the actual problem here? That `wc -l` is producing a newline on some shells? We could also just use `tr` to trim any whitespace from the grep buffer.

Side note: I realize I should've swapped `shell kind get clusters` with `shell ${KIND} get clusters`.

Reply:
The issue I saw locally was that `wc -l` was returning with extra whitespace at the front in my zsh shell:
```
rukpak(update-provisioner-name): kind get clusters | grep rukpak-e2e | wc -l
       1
```
which caused the comparison to not be accurate. There are a lot of ways to trim whitespace, I just went with the one I'm most familiar with. 

I can update to use the target to use the $KIND variable